hazelcast-jet-core/src/main/java/com/hazelcast/jet/core/DAG.java

```diff
@@ -141,10 +141,6 @@ public DAG edge(@Nonnull Edge edge) {
         if (edge.getDestination() == null) {
```

Comment:
Javadoc says: "It is an error to add an edge that connects the same two vertices as another existing edge." which is a bit confusing. This should be clarified and ref manual also updated if necessary.

Reply:
I updated this. I didn't find a good place to update the refman, I don't think it's important to mention there.